sdk/attestation/attestation/README.md

```diff
@@ -180,13 +180,27 @@ InitTime data refers to data which is used to configure the SGX enclave being at
 Creates an instance of the Attestation Client at uri `endpoint`.
 
 ```ts
-<FILL THIS IN>
+  const client = new AttestationClient(new DefaultAzureCredential(), endpoint);
+
+  // Retrieve the set of attestation policy signers from the attestation client.
+  const attestationSigners = await client.getAttestationSigners();
+
+```
+
+Creates an instance of the Attestation Administration Client at uri `endpoint`.
+
+```ts
+  const client = new AttestationAdministrationClient(new DefaultAzureCredential(), endpoint);
```

Comment:
nit: Generally, we declare the credential apart from the client.

Reply:
Silly question: Why? It takes up more vertical space and introduces a local variable that is used only once.

Is there a functional difference between:
```
const credential = new DefaultAzureCredential();
const client = new AttestationAdministrationClient(credential, endpoint);
```

and:
```
const client = new AttestationAdministrationClient(new DefaultAzureCredential(), endpoint);
```?

Maybe something related to variable scoping and closures?

Reply:
I think the credential convention is mainly to showcase that a credential is not explicitly tied to a client. In many cases, customers may want to use more than one client with one specific credential instance. For example, they may want to re-use a credential instance if they’re looking to instantiate clients on a loop since there’s some performance benefit of re-using a credential.

I believe that given that we generally put the credential outside of the client, having it here directly sent as a parameter is not an issue. Throughout the SDK, we’re not generally advocating for credentials to be instantiated once per client, so I wouldn’t assume customers would see it as necessary to tie a credential to a client. That is to say, I don’t mind if we don’t separate it here.

Reply:
Given that these are not tied to any samples, I think that updating the readme.md file to separate the two makes sense.

I'll update the readme but not the samples.